client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -24,49 +160,34 @@ class SubstanceLoadProjectMesh(load.LoaderPlugin):
     order = -10
     icon = "code-fork"
     color = "orange"
+    project_templates = []
```

Comment:
I wonder if this will help future developers reading this code:
```suggestion

    # Defined via settings
    project_templates = []
```